element_calcium_imaging/field_processing.py

```diff
@@ -372,8 +372,14 @@ def _run_motion_correction():
             params["fnames"] = [
                 f.relative_to(processed_root_data_dir).as_posix() for f in file_paths
             ]
-            pickle.dump(extra_dj_params, output_dir / f"{key['subject']}_session{key['session']}_params{key['paramset_idx']}_field{key['field_idx']}_motion_correction_results.pkl")
-            
+            motion_correction_results = extra_dj_params.pop("mc_results")
```

Comment:
I'd suggest to do the pickle dump inside the `_run_motion_correction` (at the end).
That way, no need to store and pop the `mc_results` from `extra_dj_params`. But more importantly, this way would keep the `memoized_results` robust

Reply:
Done!